src/main.cpp

```diff
@@ -382,6 +382,10 @@ std::unique_ptr<cMemorySettingsRepository> parseArguments(int argc, char **argv)
 
 		TCLAP::SwitchArg noBufArg("", "no-output-buffering", "Disable output buffering", cmd);
 
+		TCLAP::SwitchArg runAsServiceArg("d", "run-as-service", "Run as a service on Windows", cmd);
```

Comment:
If you make the first parameter an empty string and then the second parameter service, you should be able to get the old behaviour of only allowing long form.